### PR TITLE
[T] (quests) to_reach_beyond

### PR DESCRIPTION
**Pull request by nai200**
Co-authored-by: UMI <lkg20031022@outlook.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/to_reach_beyond.snbt [hunk 1]
```diff
@@ -288,23 +288,23 @@
...
+				"在搭建完成&6命令方块&r(或者&6伪命令方块&r)的结构之后，将会&b召唤凋灵风暴&r。在这样做之前，我应该&c确保做好充足的准备&r。"
 				""
-				"Killing the Wither Storm is more complex than just hitting it until it dies. In fact, it is &cinvincible until a certain point&r is reached."
+				"击杀凋灵风暴十分复杂，可不是一直攻击它直到死亡就行。事实上，它在达到&c某个特定的点之前是无敌的&r。"
 				""
-				"The Storm will &6consume nearby blocks and mobs&r, evolving over time. For the first four evolution phases, it cannot be stopped."
+				"凋灵风暴会&6吞噬附近的方块和生物&r，随着时间的推移不断进化。前四个进化阶段是无法停止的。"
 				""
-				"My goal for these phases is to simply &asurvive&r and destroy the waves of enemies that pour forth from the portals as it consumes their essence. The faster I dispatch the fiends, the faster the ritual will progress. I have to allow it to consume and evolve until it becomes confident enough to strike out... where then it may become vulnerable..."
+				"在这几个阶段中，我的目标很简单：&a活下去&r，并击败一波波从传送门中涌出的敌人，它们的灵魂已经被吞噬了。怪物召唤得越快，仪式的进展就越快。我必须让它不断地吞噬和进化，直到它变得足够强大以自主行动...那么它可能会露出它的弱点..."
```
Comment (luo-2233) [inline config/ftbquests/quests/chapters/to_reach_beyond.snbt:303]:
```suggestion
				"在这几个阶段中，我的目标很简单：&a活下去&r，并击败一波波从传送门中涌出的敌人，它们的灵魂已经被吞噬了。怪物召唤得越快，仪式的进展就越快。我必须让它不断地吞噬和进化，直到它变得足够强大以自主行动……那么它可能会露出它的弱点……"
```

File: config/ftbquests/quests/chapters/to_reach_beyond.snbt [hunk 2]
```diff
@@ -418,17 +418,17 @@
...
-			title: "The Eye of the Storm"
+			title: "风暴之眼"
 			x: -1.5d
 			y: 4.0d
 			shape: "square"
-			subtitle: "Escape the Wither Storm's gaze"
+			subtitle: "从凋灵风暴的凝视中逃脱"
 			description: [
-				"At phase 4, the Wither Storm's gaze will begin to &cpull in entities&r, including myself."
+				"在第4阶段，凋灵风暴的牵引光束会将&c实体拉向它&r,包括我自己。"
```
Comment (luo-2233) [inline config/ftbquests/quests/chapters/to_reach_beyond.snbt:427]:
```suggestion
				"在第4阶段，凋灵风暴的牵引光束会将&c实体拉向它&r，包括我自己。"
```

**Review by luo-2233 — APPROVED** [review]

File: config/ftbquests/quests/chapters/to_reach_beyond.snbt [hunk 3]
```diff
@@ -288,23 +288,23 @@
...
-				"The Storm will &6consume nearby blocks and mobs&r, evolving over time. For the first four evolution phases, it cannot be stopped."
+				"凋灵风暴会&6吞噬附近的方块和生物&r，随着时间的推移不断进化。前四个进化阶段是无法停止的。"
 				""
-				"My goal for these phases is to simply &asurvive&r and destroy the waves of enemies that pour forth from the portals as it consumes their essence. The faster I dispatch the fiends, the faster the ritual will progress. I have to allow it to consume and evolve until it becomes confident enough to strike out... where then it may become vulnerable..."
+				"在这几个阶段中，我的目标很简单：&a活下去&r，并击败一波波从传送门中涌出的敌人，它们的灵魂已经被吞噬了。怪物召唤得越快，仪式的进展就越快。我必须让它不断地吞噬和进化，直到它变得足够强大以自主行动...那么它可能会露出它的弱点..."
 				""
 				"{image:inferno:textures/quest/witherstorm.png width:250 height:124 align:1}"
 				""
-				"A phase 4 Wither Storm"
+				"一个第 4 阶段的凋灵风暴"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/to_reach_beyond.snbt:307]:
```suggestion
				"一个第4阶段的凋灵风暴"
```

File: config/ftbquests/quests/chapters/to_reach_beyond.snbt [hunk 4]
```diff
@@ -313,26 +313,26 @@
...
+			title: "腐蚀之王"
 			icon: "witherstormmod:command_block_book"
 			x: -3.0d
 			y: 6.75d
 			shape: "diamond"
-			subtitle: "Kill a Withered Symbiont"
+			subtitle: "击杀一个凋零共生体"
 			description: [
-				"At the end of evolution phase 5, the Wither Storm will summon a &bWither Symbiont&r, a giant wither-sickened zombie."
+				"在第 5 阶段的进化过程末尾，凋零风暴将会召唤 &b凋灵共生体&r，它是一只感染了凋灵病的巨大僵尸。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/to_reach_beyond.snbt:331]:
```suggestion
				"在第5阶段的进化过程末尾，凋零风暴将会召唤&b凋灵共生体&r，它是一只感染了凋灵病的巨大僵尸。"
```

File: config/ftbquests/quests/chapters/to_reach_beyond.snbt [hunk 5]
```diff
@@ -345,19 +345,19 @@
...
+			subtitle: "制造恐怖炸弹"
 			description: [
-				"The &bFormidibomb&r is the only thing that can hurt a phase 5 Wither Storm."
+				"&b恐怖炸弹&r是唯一一个可以在凋零风暴第五阶段对其造成伤害的道具。"
 				""
-				"Upon crafting, it &cimmediately begins a countdown&r, exploding at the end. I'll have to &6hurry and place it near the Storm&r during that time."
+				"制作完成之后，它会&c立即开始爆炸倒计时&r，最终爆炸。在这段时间里，我必须得&6尽快将它放在凋灵风暴附近&r。"
 				""
-				"The Formidibomb's detonation (which can be triggered with a Flint and Steel) is the only thing capable of &5killing the Wither Storm&r."
+				"恐怖炸弹的爆炸(可以用打火石触发)是唯一能够&5杀死凋零风暴&r的道具。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/to_reach_beyond.snbt:358]:
```suggestion
				"恐怖炸弹的爆炸（可以用打火石触发）是唯一能够&5杀死凋零风暴&r的道具。"
```

File: config/ftbquests/quests/chapters/to_reach_beyond.snbt [hunk 6]
```diff
@@ -345,19 +345,19 @@
...
 				""
-				"&cMAKE SURE IT IS PLACED CLOSE ENOUGH!&r I definitely do NOT want to waste these!"
+				"&c&l确保它放置的足够近！&r我绝对不会想浪费这些材料的！"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/to_reach_beyond.snbt:360]:
```suggestion
				"&c确保它放置的足够近！&r我绝对不会想浪费这些材料的！"
```

File: config/ftbquests/quests/chapters/to_reach_beyond.snbt [hunk 7]
```diff
@@ -376,29 +376,29 @@
 				{
 					id: "712DED911C63539B"
 					type: "item"
-					title: "These might come in handy."
+					title: "这些也许会派上用场."
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/to_reach_beyond.snbt:379]:
```suggestion
					title: "这些也许会派上用场。"
```

File: config/ftbquests/quests/chapters/to_reach_beyond.snbt [hunk 8]
```diff
@@ -376,29 +376,29 @@
...
 					item: "artifacts:steadfast_spikes"
 				}
 				{
 					id: "1E2CDDC305540EB0"
 					type: "toast"
-					title: "I stood my ground against the darkness."
+					title: "我在黑暗中站稳了脚跟."
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/to_reach_beyond.snbt:385]:
```suggestion
					title: "我在黑暗中站稳了脚跟。"
```

**Review by Cactusstudent — COMMENTED** [review]
修

File: config/ftbquests/quests/chapters/to_reach_beyond.snbt [hunk 9]
```diff
@@ -113,7 +113,7 @@
 				id: "1689C65749C6445F"
 				type: "command"
 				auto: "invisible"
-				command: "/tellraw @p [\"\",\"I should make sure to \",{\"text\":\"read my journal \",\"color\":\"red\"},\"before continuing.\"]"
+				command: "/tellraw @p [\"\",\"我应该确保在继续完成之前 \",{\"text\":\"阅读我的日记 \",\"color\":\"red\"},\".\"]"
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/to_reach_beyond.snbt:116]:
```suggestion
				command: "/tellraw @p [\"\",\"我应该确保在继续完成之前\",{\"text\":\"阅读我的日记\",\"color\":\"red\"},\".\"]"
```

**Review by AnzhiZhang — APPROVED** [review]

**Pull request merged**